pynq/lib/dma.py

```diff
@@ -101,7 +110,7 @@ def stop(self):
     def _clear_interrupt(self):
         self._mmio.write(self._offset + 4, 0x1000)
 
-    def transfer(self, array):
+    def transfer(self, array, start=0, nbytes=0):
```

Comment:
With PYNQ 2.5 we added support for getting valid sub-buffers with the standard `array[start:start+nbytes]` syntax. I'm wary of having multiple ways to accomplish the same thing but happy if there are advantages to a DMA-specific API.

Reply:
I don't recall seeing that array sub-buffer syntax documented (or used) anywhere.  Not that I tried too hard, as the target platform I'm using will be stuck on v2018.3 for at least another couple of months.